eng/SymbolPublishingExclusionsFile.txt

```diff
@@ -0,0 +1,10 @@
+content/LanguageServer/alpine-arm64/libe_sqlite3.so
```

Comment:
not sure all of these are strictly necessary.

Reply:
you can remove the alpine ones, we switched to the musl RID in .net8 because alpine was removed.

Reply:
Its been so long since I did the original commit, it would be good to verify that the exceptions reported in prism are no longer thrown in vscode with the changes here.

Reply:
Removed Alpine paths.